features/roomlist/impl/src/main/kotlin/io/element/android/features/roomlist/impl/components/RoomListContentView.kt

```diff
@@ -193,16 +194,27 @@ private fun RoomsViewList(
         // FAB height is 56dp, bottom padding is 16dp, we add 8dp as extra margin -> 56+16+8 = 80
         contentPadding = PaddingValues(bottom = 80.dp)
     ) {
-        when (state.securityBannerState) {
-            SecurityBannerState.RecoveryKeyConfirmation -> {
-                item {
-                    ConfirmRecoveryKeyBanner(
-                        onContinueClick = onConfirmRecoveryKeyClick,
-                        onDismissClick = { eventSink(RoomListEvents.DismissRecoveryKeyPrompt) }
-                    )
+        if (state.securityBannerState != SecurityBannerState.None) {
+            when (state.securityBannerState) {
+                SecurityBannerState.RecoveryKeyConfirmation -> {
+                    item {
+                        ConfirmRecoveryKeyBanner(
+                            onContinueClick = onConfirmRecoveryKeyClick,
+                            onDismissClick = { eventSink(RoomListEvents.DismissRecoveryKeyPrompt) }
+                        )
+                    }
                 }
+                else -> Unit
+            }
+        } else if (state.fullScreenIntentPermissionsState.shouldDisplay) {
+            item {
+                DialogLikeBannerMolecule(
```

Comment:
Maybe extract as it's done for `ConfirmRecoveryKeyBanner`, so we also can have a preview of the banner?

Reply:
https://github.com/element-hq/element-x-android/pull/3024/commits/396b75fa7b878d0bfc4ce7f960f49080c9384494